### PR TITLE
Build Views

### PR DESCRIPTION
**Pull request by jmonto55**
This pull request contains all the steps needed to build the views  for the [blog app](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md) RoR project, including:

- Implemented the design from the sneak peek wireframes for all my routes.

⚠️⚠️⚠️ IMPORTANT: if you want to populate the database automatically, run `rails db:seed` just before running `rails server` (more information in the README file), if you prefer to do it manually using `rails console`, copy and paste the data inside the `db/seeds.rb` file.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,46 @@
...
+            </div>
+          </div>
+        </card>
+        <div class="mt-8">
+          <% if post.comments.empty? %>
+            <p class="text-sm text-zinc-200 text-opacity-50 italic"> No comments yet. </p>
+          <% end %>
+          <% post.comments.each do |comment| %>
+            <p class="text-sm mt-2 ml-8"> <%= comment.author.name %>: <%= comment.text %> </p>
+          <% end %>
```
Comment (codepantha) [inline app/views/posts/index.html.erb]: - [x] I noticed you display all the comments on the posts index page. Please use your `#last_five_comments` method in your `Post` model to show the 5 most recent comments according to project requirements 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @jmonto55 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Created views consistent with wireframe 📌 
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (jmonto55) [reply app/views/posts/index.html.erb]: Thank you for the feedback 🔧

**Review by andersonlebon — APPROVED** [review]
## Hi @jmonto55 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**